src/include/enclave/enclave_util.h

```diff
@@ -138,10 +138,22 @@ extern int sgxlkl_trace_redirect_syscall;
         }                                                     \
     } while (0)
 
```

Comment:
```suggestion
/**
 * Pointer to the thread context.  This is needed because of limitations in the C address space syntax.
 */
typedef struct schedctx* threadctx_ptr;
/**
 * Pointer to the per-ethread context.
 */
static const __seg_gs threadctx_ptr *ethread_schedctx = ((__seg_gs threadctx_ptr *)48);
```